pkgs/applications/misc/kratos/default.nix

```diff
@@ -0,0 +1,45 @@
+{ fetchFromGitHub, buildGoModule, lib, stdenv }:
+
+buildGoModule rec {
+  pname = "kratos";
+  version = "v0.6.0-alpha.1.pre.2";
+
+  src = fetchFromGitHub {
+    owner = "ory";
+    repo = "kratos";
+    rev = "fa7fa701051c819ac74c919bc52721c57f662fb8";
+    sha256 = "1jyhb5cs532msqfgnhn6bnnqw6d94g3pjgirklbrysd3z4f32f8g";
+  };
+
+  vendorSha256 = "1n5g6mrr5c8z6y3ncpkap971n6m5qimpfqkq8vjgl8jkih4m282x";
+  buildPhase = ''
+    runHook preBuild
+
+    # Patch shebangs
+    files=(
+       Makefile
+       test/e2e/run.sh
+       script/testenv.sh
+       script/test-envs.sh
+       persistence/sql/migratest/update_fixtures.sh
+    )
+    for i in "''${files[@]}"; do
+      patchShebangs "$i"
+    done
+
+    # patchShebangs doesn't work for this Makefile, do it manually
+    substituteInPlace Makefile --replace '/bin/bash' '${stdenv.shell}'
+
+    GO111MODULE=on
+    make install
+
+    runHook postBuild
+  '';
+
+  meta = with lib; {
+    maintainer = with maintainers; [ mrmebelman ];
+    homepage = "https://www.ory.sh/kratos/";
+    license = licenses.asl20;
+    description = "ORY Kratos is an API-first Identity and User Management system that is built according to cloud architecture best practices.";
```

Comment:
> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does. https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes

Reply:
Removed the period and package name, thanks.